app/_how-tos/kic-proxy-cross-namespace.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Configuring Gateway API resources across namespaces
+
+description: "Route traffic to a Service in a different namespace using ReferenceGrant"
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/routing/cross-namespace/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Routing
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I route HTTP traffic to a service in a different namespace?
+  a: |
+    Set `allowedRoutes: All` on your `Gateway` resource and create a `ReferenceGrant` that allows access to Services in the current namespace for `HTTPRoute` instances from a specific namespace.
```

Comment:
```suggestion
    Set `allowedRoutes: All` on your `Gateway` resource and create a `ReferenceGrant` that allows `HTTPRoute` instances from a specific namespace to access Services in the current namespace.
```
Switched HTTPRoute and Service fragments around, it took me a couple of reads of the sentence to understand what it was saying at first.